artifacts/camara-cloudevents/event-subscription-template.yaml

```
@@ -864,6 +935,12 @@ components:
                 status: 400
                 code: INVALID_TOKEN
                 message: Only bearer token is supported
+            GENERIC_400_INVALID_SINK:
+              description: Invalid sink value
+              value:
+                status: 400
+                code: INVALID_SINK
+                message: Only https sink values are supported
```

What about more general message:
```suggestion
                message: sink not valid for the specified protocol
```

Currently we have only HTTP as event delivery protocol, but other protocols are considered by Cloudevents.
This error could be raised not only when `https` is not present, but also when the whole provided address is wrong (not resolvable).